src/kibana/plugins/kibana/_timepicker.js

```diff
@@ -24,7 +24,9 @@ define(function (require) {
       // Close if already open
       if ($scope.globalConfigTemplate === timepickerHtml) {
         delete $scope.globalConfigTemplate;
+        document.getElementById('timepicker').setAttribute('aria-expanded', 'false');
```

Comment:
not sure it's the best directive ever...but I've added one for it (ae6c541). Directives aren't something I have a ton of experience with...